File: browserify.py
#!/usr/bin/env python3


import sys

coffee_code = sys.stdin.read()
coffee_code = (
    "    exports = window.heterarchy = {}\n\n" +
    coffee_code
    .replace("require 'underscore'", "_")
    .replace("global", "window")
)
print(coffee_code)

File: script/browserify.js
#!/usr/bin/env node

fs = require('fs');

var coffee_code = fs.readFileSync('/dev/stdin').toString();
coffee_code = "    exports = window.heterarchy = {}\n\n" +
    coffee_code
    .replace("require 'underscore'", "_")
    .replace(/global/g, "window");

console.log(coffee_code);
